pkgs/args/lib/command_runner.dart

```diff
@@ -281,16 +320,24 @@ abstract class Command<T> {
   /// A single-line template for how to invoke this command (e.g. `"pub get
   /// `package`"`).
   String get invocation {
-    var parents = [name];
+    var invocation = _invocationChain;
+    if (_subcommands.containsKey(defaultSubcommand)) {
+      return '$invocation [<subcommand>] [arguments]';
```

Comment:
Is this branch covered by tests? I can't find another instance of `[<subcommand>]` in any example output.